### PR TITLE
Unified prerequisities for End-user-guide + additional fixes

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

https://github.com/eclipse/che/issues/14399

### CODE REVIEW

File: src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-https.adoc [hunk 1]
```diff
@@ -4,12 +4,21 @@
...
 .Procedure
-
 To clone a repository using HTTPS:
 
 . Use the link:https://code.visualstudio.com/docs/editor/versioncontrol#_cloning-a-repository[clone] command provided by the `vscode` Git extension.
 
-NOTE: Alternatively, use the native *Git* commands in the terminal to clone a project.
+Alternatively, use the native *Git* commands in the terminal to clone a project.
+
+.Prerequisites
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-https.adoc]: This is mixed up -- the 'Prerequisites' now come after the 'Procedure'.

File: src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-https.adoc [hunk 2]
```diff
@@ -4,12 +4,21 @@
...
+* Git is installed. Install Git if needed by following link:https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Getting Started - Installing Git]
+
+. Navigate to destination folder using `cd` command
+. Use `git clone` to clone a repository
++
+----
+$ git clone <link>
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-https.adoc]: Use italics on `<link>`.

File: src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-ssh.adoc [hunk 3]
```diff
@@ -5,9 +5,8 @@
 
 
 .Prerequisites
-
-* SSH key pair generated
-
+* Personal link:(https://help.github.com/en/articles/types-of-github-accounts)[GitHub account] created.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-ssh.adoc]: Incorrect link syntax.

File: src/main/pages/che-7/end-user-guide/proc_adding-the-vs-code-extension-using-the-workspace-configuration.adoc [hunk 4]
```diff
@@ -1,6 +1,12 @@
 [id="adding-the-vs-code-extension-using-the-workspace-configuration_{context}"]
 = Adding the VS Code extension using the workspace configuration
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_adding-the-vs-code-extension-using-the-workspace-configuration.adoc]: 'quick-starts'

File: src/main/pages/che-7/end-user-guide/proc_adding-projects-to-your-workspace.adoc [hunk 5]
```diff
@@ -3,6 +3,12 @@
 [id="adding-projects-to-your-workspace_{context}"]
 = Adding projects to your workspace
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_adding-projects-to-your-workspace.adoc]: 'quick-starts'

File: src/main/pages/che-7/end-user-guide/proc_adding-a-kubernetes-application-to-an-existing-workspace-using-the-dashboard.adoc [hunk 6]
```diff
@@ -5,6 +5,12 @@
 
 This procedure demonstrates how to modify an existing workspace and import the Kubernetes application using the newly created devfile.
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_adding-a-kubernetes-application-to-an-existing-workspace-using-the-dashboard.adoc]: 'quick-starts'

File: src/main/pages/che-7/end-user-guide/proc_downloading-and-uploading-a-file-to-a-workspace-using-the-command-line-interface.adoc [hunk 7]
```diff
@@ -7,7 +7,10 @@ This procedure describes how to use the `kubectl` tool to download or upload fil
 
 .Prerequisites
 
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
 * Remote access to the Che workspace you intend to modify. For instructions see xref:remotely-accessing-workspaces-using-the-kubernetes-command-line-tool_remotely-accessing-workspaces[].
+* The kubectl binary from the https://kubernetes.io/docs/tasks/tools/install-kubectl/[Kubernetes website].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_downloading-and-uploading-a-file-to-a-workspace-using-the-command-line-interface.adoc]:
```suggestion
* The `kubectl` binary from the https://kubernetes.io/docs/tasks/tools/install-kubectl/[Kubernetes website].
```

File: src/main/pages/che-7/end-user-guide/proc_downloading-and-uploading-a-file-to-a-workspace-using-the-command-line-interface.adoc [hunk 8]
```diff
@@ -7,7 +7,10 @@ This procedure describes how to use the `kubectl` tool to download or upload fil
...
+* To verify the installation of `kubectl` using the `kubectl version` command:
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_downloading-and-uploading-a-file-to-a-workspace-using-the-command-line-interface.adoc]:
```suggestion
* Verify the installation of `kubectl` using the `kubectl version` command:
```

File: src/main/pages/che-7/end-user-guide/proc_exporting-a-workspace-with-a-che-factory.adoc [hunk 9]
```diff
@@ -1,23 +1,16 @@
 [id="exporting-a-workspace-with-a-che-factory_{context}"]
 = Exporting a workspace with a Che factory
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+
 
 [discrete]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_exporting-a-workspace-with-a-che-factory.adoc]:
```suggestion
```

File: src/main/pages/che-7/end-user-guide/proc_exporting-a-workspace-with-a-che-factory.adoc [hunk 10]
```diff
@@ -1,23 +1,16 @@
...
 .Procedure
 
-. Start each step with an active verb.
-
-. Include one command or action per step.
 
-. Use an unnumbered bullet (*) if the procedure includes only one step.
 
 [discrete]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_exporting-a-workspace-with-a-che-factory.adoc]:
```suggestion
```

File: src/main/pages/che-7/end-user-guide/proc_exporting-a-workspace-with-a-che-factory.adoc [hunk 11]
```diff
@@ -1,23 +1,16 @@
...
+
 
 [discrete]
 .Prerequisites
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
 
 [discrete]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_exporting-a-workspace-with-a-che-factory.adoc]:
```suggestion
```

File: src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-open-button.adoc [hunk 12]
```diff
@@ -3,15 +3,13 @@
...
 This section describes how to run an existing workspace from the User Dashboard using the *Open* button.
 
-[discrete]
-== Prerequisites
+.Prerequisites
 
 * A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
 
-* An existing workspace defined on this instance of Eclipse Che. 
+* An existing workspace defined on this instance of Eclipse Che xref:creating-a-workspace-from-user-dashboard_{context}[Creating a workspace from user dashboard].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-open-button.adoc]:
```suggestion
* An existing workspace defined on this instance of Eclipse Che. See xref:creating-a-workspace-from-user-dashboard_{context}[].
```

Comment (rkratky) [reply src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-open-button.adoc]: Does this `xref` work? (Is the link on the same page?)

File: src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-recent-workspaces.adoc [hunk 13]
```diff
@@ -3,15 +3,12 @@
 
 This section describes how to run an existing workspace from the User Dashboard using the Recent Workspaces.
 
-[discrete]
-== Prerequisites
+.Prerequisites
 
 * A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
+* An existing workspace defined on this instance of Eclipse Che.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-recent-workspaces.adoc]: Why inconsistent? (No link to instructions.)

File: src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-run-button.adoc [hunk 14]
```diff
@@ -3,14 +3,12 @@
 
 This section describes how to run an existing workspace from the User Dashboard using the *Run* button.
 
-[discrete]
-== Prerequisites
+.Prerequisites
 
 * A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
+* An existing workspace defined on this instance of Eclipse Che xref:creating-a-workspace-from-user-dashboard_{context}[Creating a workspace from user dashboard].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-run-button.adoc]:
```suggestion
* An existing workspace defined on this instance of Eclipse Che. See xref:creating-a-workspace-from-user-dashboard_{context}[].
```

Comment (rkratky) [reply src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-run-button.adoc]: Does this `xref` work? (Is the link on the same page?)

File: src/main/pages/che-7/end-user-guide/proc_running-and-debugging.adoc [hunk 15]
```diff
@@ -5,8 +5,10 @@
 
 Che-Theia supports the link:https://microsoft.github.io/debug-adapter-protocol/[Debug Adapter Protocol]. This protocol defines a generic way for how a development tool can communicate with a debugger. It means Che-Theia works with all link:https://microsoft.github.io/debug-adapter-protocol/implementors/adapters/[implementations].
 
-.Procedure
+.Prerequisites
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_running-and-debugging.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_including-a-kubernetes-application-in-a-workspace-devfile-definition.adoc [hunk 16]
```diff
@@ -3,6 +3,14 @@
 [id="including-a-kubernetes-application-in-a-workspace-devfile-definition_{context}"]
 = Including a Kubernetes application in a workspace devfile definition
 
+This procedure demonstrates how to define the Che 7 workspace devfile by Kubernetes application.
+
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_including-a-kubernetes-application-in-a-workspace-devfile-definition.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_importing-a-vs-code-extension-in-the-che-theia-editor.adoc [hunk 17]
```diff
@@ -7,6 +7,8 @@ This method of installing VS Code extensions is for extensions that do not have
 
 .Prerequisites
 
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_importing-a-vs-code-extension-in-the-che-theia-editor.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_generating-a-devfile-from-an-existing-kubernetes-application.adoc [hunk 18]
```diff
@@ -9,6 +9,8 @@ This procedure demonstrates how to generate a devfile from an existing Kubernete
 
 .Prerequisites
 
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_generating-a-devfile-from-an-existing-kubernetes-application.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_downloading-and-uploading-a-file-to-a-workspace-using-the-command-line-interface.adoc [hunk 19]
```diff
@@ -7,7 +7,10 @@ This procedure describes how to use the `kubectl` tool to download or upload fil
 
 .Prerequisites
 
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_downloading-and-uploading-a-file-to-a-workspace-using-the-command-line-interface.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-by-importing-source-code-of-a-project.adoc [hunk 20]
```diff
@@ -15,6 +15,12 @@ summary:
 
 This section describes how to create a new workspace to edit an existing codebase.
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-by-importing-source-code-of-a-project.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-by-importing-source-code-of-a-project.adoc [hunk 21]
```diff
@@ -15,6 +15,12 @@ summary:
...
+
+* A workspace with plug-ins related to your development environment on this instance of Eclipse Che xref:creating-a-workspace-from-user-dashboard_{context}[Creating a workspace from user dashboard].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-by-importing-source-code-of-a-project.adoc]: Does this `xref` work? (Is the link on the same page?)

Comment (rkratky) [reply src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-by-importing-source-code-of-a-project.adoc]: When using an xref, the title of the link is filled in automatically.

File: src/main/pages/che-7/end-user-guide/proc_creating-a-new-workspace-from-the-dashboard.adoc [hunk 22]
```diff
@@ -5,6 +5,10 @@
 
 This procedure describes how to create and edit a new Che 7 devfile using the *Dashboard*.
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_creating-a-new-workspace-from-the-dashboard.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc [hunk 23]
```diff
@@ -5,6 +5,12 @@
 
 This section describes how to convert a Che 6 workspace to a Che 7 devfile. The result is a basic Che 7 devfile that can be used for further workspace creation.
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc [hunk 24]
```diff
@@ -5,6 +5,12 @@
...
+
+* An existing workspace defined on this instance of Eclipse Che xref:creating-a-workspace-from-user-dashboard_{context}[Creating a workspace from user dashboard].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc]: Does this `xref` work? (Is the link on the same page?)

File: src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc [hunk 25]
```diff
@@ -30,6 +36,12 @@ image::workspaces/adding-pluggins-to-workspace.png[link="{imagesdir}/workspaces/
 
 == Defining the workspace editor
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc]:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

File: src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc [hunk 26]
```diff
@@ -18,6 +18,12 @@ Che 7 plug-ins replace Che 6 installers. The following table lists the Che 7 plu
 | org.eclipse.che.ls.java  | Language Support for Java
 |===
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc]: Same old, same old... See the other comments and suggestions about these prereqs.

**Review by rkratky — DISMISSED** [review]

**Review by themr0c — APPROVED** [review]
LGTM

**Pull request merged**